### ISSUE TITLE
Error with Multiple Bot Credentials 

### ISSUE THREAD

**Issue by mzrimsek** [edbrdi/n8n-nodes-discord#22]
I have two Discord bots added as credentials to n8n. I've verified that this node works with both of them individually (so I know the credentials are correct).

The issue I am seeing is when you select a discord bot credential for the node, you list of channels and roles will get stuck and will not refresh if you change the credential. For instance I select bot one credentials and get the appropriate list of channels to listen to. If I then select my other bot credentials (which is part of a different server) and try to select a channel, the list is still the channels for the first credentials.

Closing the node and reopening does not fix it. Deleting the node and creating a new one does not fix it. The only thing I've found that fixes it is restarting the container altogether.

If this is the limitation of the node that's fine, I just think it should be documented if it's intended behavior because based on the docs it seems like this is a bug.

I was hoping to move a few bots I am running just into a few workflows to reduce maintenance.

**Comment by edbrdi** [comment]
Yes, it's a node limitation, not intended for use with multiple bots. I'll update the doc accordingly.

**Comment by mzrimsek** [comment]
Much appreciated!

**Comment by pfordmedia** [comment]
Having the same issue actually. Is there a way to avoid the OAuth2 flow and add in the ability to add headers? It would just use a bot token. Im using this node for presence updates as I can't figure out a way to do it with an API call using HTTP node...
